cmd/utils.go

```diff
@@ -114,13 +114,16 @@ func GetOutdatedPackages(installed map[string]desc.Desc, availablePackages cran.
 	for _, pkgDl := range availablePackages.Packages {
 
 		pkgName := pkgDl.Package.Package
-
-		if _, found := installed[pkgName]; found {
-			outdatedPackages = append(outdatedPackages, gpsr.OutdatedPackage {
-				Package:    pkgName,
-				OldVersion: installed[pkgName].Version,
-				NewVersion: pkgDl.Package.Version,
-			})
+		availableVersion := pkgDl.Package.Version
+
+		if installedPkg, found := installed[pkgName]; found {
+			if availableVersion != installedPkg.Version {
```

Comment:
this still should use the desc version comparison, however this has been fixed upstream already in add-update feature so will not fix it now since that branch should merge quickly after